tasks/Module 4/task_6.py

```diff
@@ -0,0 +1,20 @@
+"""Brute-Force PDF Password Breaker."""
+import os
+import PyPDF2
```

Comment:
Usually we add empty lines between groups (built-in, 3rd party, your own) of imports. So
```python
import os

import PyPDF2
```
or in more complicated case:
```python
import csv
import os
import sys

import openpyxl
import PyPDF2

import your_own_awesome_module_from_same_project
```